packages/server/tsconfig.json

```diff
@@ -2,22 +2,22 @@
   "compilerOptions": {
     "module": "commonjs",
     "esModuleInterop": true,
-    "target": "es2017",
+    "target": "es5",
```

Comment:
You are going to use latest Node.js, so you don't need to compile your code to ancient ES5.